### DISCUSSION TITLE
Forward local private SSH keys

### DISCUSSION THREAD

**Discussion by keichigo** [community/community#66294] [Codespaces] [answered]
### Select Topic Area

Question

### Body

On the other hand, if I use vscode-remote to a server via the Remote-SSH extension, the keys are forwarded correctly.

Forwarding private SSH keys would be useful to be able to access dependencies hosted in private Github repos (among other things).

Why my SSH keys are not forwarded also to the Codespaces?

**Comment by mehulrajdev** [comment, marked as answer]
Posted at https://github.com/microsoft/vscode-remote-release/issues/8816
